lib/decorators/api-extension.decorator.ts

```diff
@@ -9,10 +9,7 @@ export function ApiExtension(extensionKey: string, extensionProperties: any) {
   }
 
   const extensionObject = {
-    [extensionKey]:
-      typeof extensionProperties !== 'string'
-        ? { ...extensionProperties }
-        : extensionProperties
+    [extensionKey]: extensionProperties
```

Comment:
```suggestion
    [extensionKey]: clone(extensionProperties)
```

Where `clone` is imported from `lodash`.